docker/assets/grafana/dashboards/analytics_dashboard.json

```diff
@@ -2027,7 +2027,7 @@
                         [
                             {
                                 "params": [
-                                    "claimed_value"
+                                    "unclaimed_value"
```

Comment:
Please also change the aliases accordingly.

Reply:
Good call, thanks.